src/main/java/org/isf/exa/service/CommonExamIOOperations.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
I don't see the point of this super class given that the code references two different repositories.   I don't agree with this change either.